src/core/utils/index.js

```diff
@@ -336,3 +348,39 @@ export const getAddonTypeFilter = (addonType, _config = config) => {
 
   return ADDON_TYPE_THEMES_FILTER;
 };
+
+export const getExtensionTypeAction = (type = ADDON_TYPE_EXTENSION) => {
```

Comment:
This seems to duplicate the logic also found in `getAction` in `src/core/tracking.js`. Is there a need for both functions?

Also, I think the whole function could be reduced to:

```
export const getExtensionTypeAction = () => {
  return type === ADDON_TYPE_STATIC_THEME
    ? TRACKING_TYPE_STATIC_THEME
    : TRACKING_TYPE_EXTENSION;
}
```

Also, `getExtensionTypeAction` doesn't seem like an accurate description. I think this is more like `getTrackingAddonType`.

Reply:
oh?! i didn't not see the `getAction`. I'll take a look...

Regarding the name tho, it's not really an add on type  - right now it's either "addon" or "theme". But i'll revisit this name if it is in fact needed